### PR TITLE
feat(watt): add icon button to Design System

### PR DESCRIPTION
**Pull request by dzhavat**
### Design System

- Add separate `<watt-icon-button>` component
- Expose the component from `WattButtonModule`
- Adjust Storybook to use the new component
- Fix property values in Storybook overview page

## References

- https://github.com/Energinet-DataHub/MightyDucks/issues/215

## PLEASE NOTE

Before merging a frontend pull request, be sure that all checks have passed. Currently, jobs done by a Git-bot (like formatting and adding licenses) don't trigger a re-run of all the other checks. The "merge button" can therefore give you a false positive signal.

### CODE REVIEW

**Comment by dzhavat** [comment]
clarification: The reason for creating a separate component is because I didn't see how the existing `watt-button` could support an icon only and not end up with a `min-width` of 100px. One solution was to add an extra `@Input` property, e.g. `iconOnly` (boolean), to specifically that the button is just an icon. I didn't go that way because the `watt-button` already has many `@Input` properties and adding more would've made it more complicated to keep track of them at the same time.

File: libs/ui-watt/src/lib/components/button/watt-button.stories.ts [hunk 1]
```diff
@@ -63,3 +69,24 @@ export const Button = ButtonTemplate.bind({});
 Button.args = {
   type: 'text',
 };
+
+const IconButtonTemplate: Story<WattIconButtonComponent> = (args) => ({
+  props: args,
+  template: `<watt-icon-button icon="${args.icon}" [disabled]="${args.disabled}"></watt-icon-button>`,
```
Comment (frontend-specialisten) [inline libs/ui-watt/src/lib/components/button/watt-button.stories.ts:75]: This template doesn't seem to be reflected in the docs:
image

**Review by frontend-specialisten — CHANGES_REQUESTED** [review]
Couldn't the icon button just be a type like text? Otherwise it looks good to me, only a single comment about the documentation example :)

**Comment by dzhavat** [comment]
> Couldn't the icon button just be a type like text? Otherwise it looks good to me, only a single comment about the documentation example :)

Do you mean adding a fourth type `icon` to `WattButtonType`?

File: libs/ui-watt/src/lib/components/button/watt-button.stories.ts (hunk 1 above)
Comment (dzhavat) [reply libs/ui-watt/src/lib/components/button/watt-button.stories.ts]: Fixed.

**Review by frontend-specialisten — APPROVED** [review]

**Pull request merged**